<launch>

    joystick node
    <node name="joy" pkg="joy" type="joy_node" required="true" >
        <param name="~dev"                  value="/dev/input/js0" />
        <param name="~deadzone"             value="0.01" />
    </node>
    
    <!-- <rosparam file="$(find bimanual_teleop_controller)/config/pr2_joint_group_velocities_controller.yaml" command="load" /> -->
    <!-- <node pkg="pr2_controller_manager" type="spawner" name="spawner" args="pr2_joint_group_vel_controller" output="screen"/>
    <node pkg="pr2_controller_manager" type="unspawner"  name="unspawner_r_arm_controller" args="l_arm_controller" output="screen"/>
    <node pkg="pr2_controller_manager" type="unspawner"  name="unspawner_l_arm_controller" args="r_arm_controller" output="screen"/> -->
    <node pkg="bimanual_teleop_controller" type="pr2_gazebo_vel_single_joint.py" name="pr2_vel_control" output="screen"/>

</launch>